docker/Dockerfile.ingest

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 Amazon.com, Inc. or its affiliates.
+
+FROM public.ecr.aws/lambda/python:3.11 as osml_data_ingest
+
+RUN yum -y upgrade
+RUN yum -y install gcc
+RUN yum -y install ca-certificates
+
+RUN pip3 install stac_fastapi.opensearch stac_fastapi.types
+
+# Copy the function code to the LAMBDA_TASK_ROOT directory
+# This environment variable is provided by the lambda base image
+ADD . ${LAMBDA_TASK_ROOT}
+
+CMD ["src/aws/osml/data_intake/processors/ingest_processor.handler"]
```

Comment:
Could you do `pip3 install .` (one directory up)

Then you can update the cmd to be like

```
CMD ["aws.osml.data_intake.processors.ingest_processor.handler"]
```